src/rpc/blockchain.cpp

```diff
@@ -2275,6 +2377,7 @@ static const CRPCCommand commands[] =
     { "blockchain",         "preciousblock",          &preciousblock,          {"blockhash"} },
     { "blockchain",         "scantxoutset",           &scantxoutset,           {"action", "scanobjects"} },
     { "blockchain",         "getblockfilter",         &getblockfilter,         {"blockhash", "filtertype"} },
+    { "blockchain",         "dumptxoutset",           &dumptxoutset,           {"path"} },
```

Comment:
In commit "rpc: add dumptxoutset" (b6af8f5443c9891ade4fa5ca92a244e3ff8cac79)

Might want to move this to the hidden section, unless it's useful for something other than debugging. Also, if the format or usage might change, it might be better to not expose so compatibility is less a concern.